int32 state
int32 trial_number
int32 nTrials
int32 trial_length
int32 set_type
int32 task
---

